### PR TITLE
gnome3.pomodoro: 0.13.4 -> 0.15.1

### PR DESCRIPTION
**Pull request by worldofpeace**
###### Motivation for this change
Fix build because it's failing in staging-next https://hydra.nixos.org/build/91945945
due to the vala bump. Then noticed it was pretty out of date.

This brings in a new package [Gom](https://wiki.gnome.org/Projects/Gom).
I had to make an MR to it because of the PyGObject override
* https://gitlab.gnome.org/GNOME/gom/merge_requests/3

###### Things done
Built and executed, haven't tested the shell extension though.

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by worldofpeace** [comment]
This [test](https://gitlab.gnome.org/GNOME/gom/blob/master/tests/meson.build#L28) reached timeout on ofborg. Maybe it won't be a problem on hydra :thinking:

File: pkgs/desktops/gnome-3/misc/pomodoro/default.nix [hunk 1]
```diff
@@ -1,49 +1,73 @@
...
+    gtk3
+    libcanberra
+    libpeas
+    sqlite
   ];
 
   meta = with stdenv.lib; {
-    homepage = http://gnomepomodoro.org/;
+    homepage = https://gnomepomodoro.org/;
     description = "A time management utility for GNOME based on the pomodoro technique";
```
Comment (jtojnar) [inline pkgs/desktops/gnome-3/misc/pomodoro/default.nix]:
```suggestion
    description = "Time management utility for GNOME based on the pomodoro technique";
```

**Comment by jtojnar** [comment]
Hmm, I am encountering

https://gitlab.gnome.org/GNOME/gom/issues/22

**Comment by worldofpeace** [comment]
> Hmm, I am encountering
> 
> https://gitlab.gnome.org/GNOME/gom/issues/22

ofborg was too. I could bump the timeout to 500 if that sounds reasonable?

**Review by bobvanderlinden — APPROVED** [review]
I was able to build and run this. :+1:

**Pull request merged**